src/main/windows/callsWidgetWindow.ts

```diff
@@ -403,6 +404,11 @@ export class CallsWidgetWindow {
             }
         }
 
+        if (!await PermissionsManager.doPermissionRequest(view.webContentsId, 'screenShare', {requestingUrl: view.view.server.url.toString(), isMainFrame: false})) {
```

Comment:
Is this going to show when you start a screenshare or calls?

Reply:
Just a screen share.

Reply:
And the permission is granted once or does it ask each time?

Reply:
It's granted once, same as all of the other permissions